data/locale/fr-FR.ini

```diff
@@ -4,22 +4,22 @@ NDIPlugin.SourceProps.SourceName="Source"
 NDIPlugin.SourceProps.Bandwidth="Bandwidth"
 NDIPlugin.SourceProps.Sync="Synchronisation"
 NDIPlugin.SourceProps.HWAccel="Autoriser l'accélération matérielle"
-NDIPlugin.SourceProps.AlphaBlendingFix="Correction du compositing alpha (ajoute un filtre à la source)"
+NDIPlugin.SourceProps.AlphaBlendingFix="Correction de simulation de transparence (ajoute un filtre à la source)"
```

Comment:
This translates back in to English as:
> Transparency simulation fix (adds filter to source)

No mention of "Alpha"; Is that acceptable?